effekt/shared/src/main/scala/effekt/Typer.scala

```diff
@@ -829,11 +830,22 @@ object Typer extends Phase[NameResolved, Typechecked] {
           Result((), effBinding)
         }
 
-      case d @ source.ExternDef(pure, id, tps, vps, bps, tpe, body) => Context.withUnificationScope {
+      case d @ source.ExternDef(captures, id, tps, vps, bps, tpe, bodies) => Context.withUnificationScope {
         d.symbol.vparams foreach Context.bind
         d.symbol.bparams foreach Context.bind
 
-        body.args.foreach { arg => checkExpr(arg, None) }
+        flowingInto(Context.lookupCapture(id.symbol.asBlockSymbol)) {
```

Comment:
would the following work?
```suggestion
        flowingInto(Context.lookupCapture(d.symbol)) {
```

Reply:
Should work, will change.